plugins/gpu/nvidia_gpu_

```diff
@@ -87,95 +86,110 @@ if [ "$1" = "suggest" ]; then
 	echo "mem"
 	echo "fan"
 	echo "power"
+	echo "utilization"
 	exit 0
 fi
 
 # Get number of GPUs
-nGpusOutput=`$nvSmiExec -L`
-nGpus=`echo "$nGpusOutput" | wc -l`
-if [ $nGpus -eq 0 ]; then
+nGpusOutput=$($nvSmiExec -L)
```

Comment:
You should quote `$nvSmiExec` here, too, I guess?
Or could it be a multi-token thing?